src/ConnectedMode/Binding/UnintrusiveBindingPathProvider.cs

```diff
@@ -30,14 +30,16 @@ namespace SonarLint.VisualStudio.ConnectedMode.Binding
     internal interface IUnintrusiveBindingPathProvider
     {
         string Get();
+
+        string SLVSRootBindingFolder { get; }
     }
 
     [Export(typeof(IUnintrusiveBindingPathProvider))]
     internal class UnintrusiveBindingPathProvider : IUnintrusiveBindingPathProvider
     {
         private readonly ISolutionInfoProvider solutionInfoProvider;
 
-        private readonly string SLVSRootBindingFolder;
+        public string SLVSRootBindingFolder { get; }
```

Comment:
+1 point to the suggestion of merging this class with the new one

Reply:
We'll need this info anyway even with the new class